compiler/src/iree/compiler/Codegen/Common/GPU/GPUHeuristics.h

```diff
@@ -30,6 +36,18 @@ struct GPUMatmulShapeType {
                      Type b, Type c)
       : mSizes(m), nSizes(n), kSizes(k), batchSizes(batch), aType(a), bType(b),
         cType(c) {}
+
+  // Constructor with the num_rhs parameter.
```

Comment:
This is not super helpful IMO and the variable name in the comment doesn't match the code.